index.html

```diff
@@ -264,27 +264,26 @@
 enum ParityType {
   "none",
   "even",
-  "mark",
   "odd",
+  "mark",
   "space"
 };
 </pre>
 <dl data-dfn-for"ParityType">
-  <dt>
-    none
-  </dt>
-  <dt>
-    even
-  </dt>
-  <dt>
-    mark
-  </dt>
-  <dt>
-    odd
-  </dt>
-  <dt>
-    space
-  </dt>
+  <dt><dfn>none</dfn></dt>
+  <dd>No parity bit is sent for each data word</dd>
+
+  <dt><dfn>even</dfn></dt>
+  <dd>Data word plus parity bit must have even parity</dd>
```

Comment:
[Statement of fact](https://ln.hixie.ch/?start=1140242962&count=1) here too. 

```suggestion
  <dd>Data word plus parity bit has even parity.</dd>
```